core/server/worker/src/main/java/alluxio/worker/grpc/ShortCircuitBlockReadHandler.java

```diff
@@ -140,7 +140,11 @@ public void onCompleted() {
       @Override
       public OpenLocalBlockResponse call() throws Exception {
         if (mLockId != BlockLockManager.INVALID_LOCK_ID) {
-          mWorker.unlockBlock(mLockId);
+          try {
+            mWorker.unlockBlock(mLockId);
+          } catch (BlockDoesNotExistException e) {
+            LOG.warn("Failed to unlock lock {} with error {}.", mLockId, e.getMessage());
```

Comment:
For all the locks, by convention, which do we log, lockid or block id?
Better to log in the same way

Reply:
I think we can print both `lockId` and `blockId`, `blockId` let us know the affect blocked block and file, `lockId` make it possible to unlock the bad lock when we have a global maintenance able lock view.